supabase/migrations/20240530291033_add_completed_parcels_view.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE VIEW completed_parcels AS
+WITH completed_events AS (
+    SELECT
+        events.parcel_id,
+        MAX(events.timestamp) AS completed_timestamp
+    FROM
+        events
+    WHERE
+        new_parcel_status IN (
+            SELECT
+                event_name
+            FROM
+                status_order
+            WHERE
+                is_successfully_completed_event
+        )
```

Comment:
Think you can remove the new `is_successfully_completed_event` column and just use 
```
WHERE new_parcel_status LIKE 'Parcel Collected' or new_parcel_status LIKE 'Fulfilled with Trussell Trust' or new_parcel_status like 'Delivered'
```

Reply:
Yep I definitely can. I had it like that before, though spoke with @AmeliaIsCoding about this and she made the point that having it as a separate column means that we can more easily edit what status' count as completed. E.g if they start working with a new trust, it's less likely to be missed off the list and unnoticed. Am happy to switch it back though if you disagree with this :)

If so - would `WHERE new_parcel_status IN ('Parcel Collected', 'Fulfilled with Trussel Trust', 'Delivered')` be a good way of writing that? Thanks!

Reply:
Ah that's a pretty good point, let's leave it